# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/string.h.patch
# Copyright (C) 2006 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Those index crap breaks applications, including GCC's libstdc++ !!!

  - Rene Rebe <rene@exactcode.de>

--- ./include/string.h.orig	2006-12-14 17:05:59.000000000 +0100
+++ ./include/string.h	2006-12-14 17:10:22.000000000 +0100
@@ -330,8 +330,10 @@
 #  define bcopy(src,dest,n) (memmove((dest), (src), (n)), (void) 0)
 #  define bzero(s,n) (memset((s), '\0', (n)), (void) 0)
 #  define bcmp(s1,s2,n) memcmp((s1), (s2), (size_t)(n))
+#if 0
 #  define index(s,c) strchr((s), (c))
 #  define rindex(s,c) strrchr((s), (c))
+#endif
 #  endif
 # endif
 
